### PR TITLE
Feature mu sql table

### PR DESCRIPTION
**Pull request by fedescarpa**
Fixes https://github.com/mumuki/mumuki-sqlite-runner/issues/4

### CODE REVIEW

**Comment by flbulgarelli** [comment]
Could you please add some screenshots?

**Comment by flbulgarelli** [comment]
Also, is it possible to include the mumuki-styles into the README, so that we can see how the component looks like?

Or at least, could we add an screenshot for each component in the readme?

File: index.html [hunk 1]
```diff
@@ -80,49 +85,14 @@
...
-          "type": "Integer",
-          "pk": true
-        },
-        "ent2_id": {
-          "type": "Integer",
-          "pk": true,
-          "fk": {
-            "to": { "entity": "Entity_2", "column": "ent2_id" },
-            "type": "many_to_one"
-          }
```
Comment (flbulgarelli) [inline index.html:119]: Why this change?

**Review by flbulgarelli — CHANGES_REQUESTED** [review]
Could we update the README with usage instructions and screenshots, too?

File: index.html [hunk 2]
```diff
@@ -80,49 +85,14 @@
...
-      }
-    }'>
+  <div class='mu-sql-table'
+    data-name='Personas'
+    data-columns='[{"name": "id_persona", "pk": true}, "Nombre", "Apellido", {"name": "Pareja", "fk": true}]'
+    data-rows='[
+      [1, "Homero", "Simpson", 2],
+      [2, "Marge", "Bouvier", 1],
+      [3, "Moe", "Szyslak", null]
+    ]'>
```
Comment (flbulgarelli) [inline index.html:95]: I really like this format when describing a new table from scratch. 

However, if I already have a table, I think it is cumbersome to convert if to this format. 

Could we support an alternate and complementary format where the table is created using an standard `<table>`? 

Something like this: 

```
<table class='mu-sql-table' data-name='Personas'>
  <th>
    <td data-pk="true" >id_persona</td>
    <td>nombre</td>
    <td>apellido</td>
    <td data-fk="true">pareja</td>
  </th>
  <tr>
    <td>1</td>
    <td>"Homero"</td>
    <td>"Simpson"</td>
    <td>2</td>
  </tr>
  <tr>
    <td>2</td>
    <td>"Marge"</td>
    <td>"Bouvier"</td>
    <td>1</td>
  </tr>
  <tr>
    <td>3</td>
    <td>"Moe"</td>
    <td>"Szyslak"</td>
    <td>n</td>
  </tr>
</table>
```

Comment (fedescarpa) [reply index.html:95]: We need a header to put the Entity name.

if you write the table into a div work fine too:

```html
<div class="mu-sql-table-rendered">
  <header>Personas</header>
  <table>
    <thead>
      <tr>
        <th class="mu-sql-table-pk">id_persona</th>
        <th>Nombre</th>
        <th>Apellido</th>
        <th class="mu-sql-table-fk">Pareja</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>1</td>
        <td>Homero</td>
        <td>Simpson</td>
        <td>2</td>
      </tr>
      <tr>
        <td>2</td>
        <td>Marge</td>
        <td>Bouvier</td>
        <td>1</td>
      </tr>
      <tr>
        <td>3</td>
        <td>Moe</td>
        <td>Szyslak</td>
        <td>NULL</td>
      </tr>
    </tbody>
  </table>
</div>
```

Comment (flbulgarelli) [reply index.html:95]: Great! Then please also document this way of defining a table :smile:

Comment (fedescarpa) [reply index.html:95]: That HTML render this table

image

File: index.html (hunk 1 above)
Comment (fedescarpa) [reply index.html:119]: because it's twice, I removed one of both

**Review by flbulgarelli — APPROVED** [review]
